test/game/TestGame.cpp

```diff
@@ -0,0 +1,185 @@
+#include <boost/test/unit_test.hpp>
+
+#include "ClientAppFixture.h"
+#include "Empire/Empire.h"
+#include "universe/Enums.h"
+#include "util/Directories.h"
+#include "util/Process.h"
+#include "util/SitRepEntry.h"
+
+namespace {
+    std::string ServerClientExe() {
```

Comment:
`ServerClient`

excuse me, what?

Reply:
I've got it from `client/human/HumanClientApp.cpp:413`